packages/docs/fluent-editor/demos/ai.vue

```
@@ -0,0 +1,40 @@
+<script setup lang="ts">
+import type FluentEditor from '@opentiny/fluent-editor'
```

ESLint

[error] 'FluentEditor' is defined but never used.

(unused-imports/no-unused-imports)